### PR TITLE
Enable distributed tracing by default

### PR DESCRIPTION
**Pull request by brettlangdon**
Enable distributed tracing by default in the following integrations:

- aiohttp
- bottle
- flask
- molten
- pylons
- pyramid
- requests
- tornado

### CODE REVIEW

**Comment by brettlangdon** [comment]
@palazzem I took a minimal approach to documentation changes, but wondering if instead I should just remove/rework the parts talking about distributed tracing. Thoughts?

File: ddtrace/contrib/falcon/__init__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
     from ddtrace import tracer
     from ddtrace.contrib.falcon import TraceMiddleware
 
-    mw = TraceMiddleware(tracer, 'my-falcon-app', distributed_tracing=True)
+    mw = TraceMiddleware(tracer, 'my-falcon-app')
```
Comment (palazzem) [inline ddtrace/contrib/falcon/__init__.py]: 👍 better to remove these kwargs from docs and leverage env vars or `Config` API. Eventually we can start deprecating / removing old ways of configuring our integrations.

File: ddtrace/contrib/molten/__init__.py [hunk 2]
```diff
@@ -13,8 +13,8 @@ def hello(name: str, age: int) -> str:
 
     ddtrace-run python app.py
 
-To enable distributed tracing when using autopatching, set the
-``DD_MOLTEN_DISTRIBUTED_TRACING`` environment variable to ``True``.
+To disable distributed tracing when using autopatching, set the
```
Comment (palazzem) [inline ddtrace/contrib/molten/__init__.py]: In general I think we should not advice how to disable distributed tracing. But to move forward, I would say:
1. change it ON by default (this PR)
2. Have @andrewsouthard1 to look at our docs and give a huge revamp

**Comment by brettlangdon** [comment]
I updated documentation to remove explicit references for "disabling distributed tracing".

**Pull request merged**